src/ucp/wireup/address.c

```diff
@@ -86,7 +86,7 @@ typedef struct {
 typedef struct {
     ucp_rsc_index_t  rsc_index;
     float            lat_ovh;
-} ucp_address_unified_iface_attr_t;
+} UCS_S_PACKED ucp_address_unified_iface_attr_t;
```

Comment:
is this passed on the wire?

Reply:
yes, it is packed here:
https://github.com/openucx/ucx/blob/95894124c59b7d2824984699225af18b0dd20fbb/src/ucp/wireup/address.c#L392